app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -39,7 +39,7 @@ class ProductTests {
     }
 
     @After
-    fun TearDown(){
+    fun tearDown(){
```

Comment:
Change made just based on best practices with Camel Case.

Reply:
Duplicates a previous review.  When that happens, first in wins.